### PR TITLE
Rename get lsig

### PR DESCRIPTION
**Pull request by robert-zaremba**
Following discussion on Discord, the `getLogicSig` is not very clear. So updating the comment and renaming it. It's a breaking change unfortunately.

### CODE REVIEW

File: examples/bond/test/bond-token-flow.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ describe('Bond token Tests', function () {
       TMPL_APP_MANAGER: appManager.address
     };
     const issuerLsigProg = getProgram('issuer-lsig.py', scInitParam);
-    lsig = runtime.getLogicSig(issuerLsigProg, []);
+    lsig = runtime.createLSigAccount(issuerLsigProg, []);
```
Comment (ratik21) [inline examples/bond/test/bond-token-flow.js:110]: just to confirm, `LSig` <--> `Lsig`.

File: packages/algob/src/lib/lsig.ts [hunk 2]
```diff
@@ -14,8 +14,8 @@ import { CompileOp } from "./compile";
 export async function getLsig (
   name: string,
   algodClient: Algodv2,
-  scTmplParams?: SCParams):
-  Promise<LogicSigAccount> {
+  scTmplParams?: SCParams): Promise<LogicSigAccount> {
+
```
Comment (ratik21) [inline packages/algob/src/lib/lsig.ts:18]: could you please fix the failing github action?

File: packages/runtime/src/runtime.ts [hunk 3]
```diff
@@ -558,11 +558,14 @@ export class Runtime {
   }
 
   /**
-   * Returns logic signature
+   * Creates a new account with logic signature and smart contract arguments
+   * in the runtime store. The arguments are use when we send a transaction with this
+   * account and verify it.
```
Comment (ratik21) [inline packages/runtime/src/runtime.ts:563]:
```suggestion
   * Creates a new account with logic signature and smart contract arguments
   * in the runtime store. The arguments are used when we send a transaction with this
   * account and verify it.
```

File: packages/runtime/src/runtime.ts [hunk 4]
```diff
@@ -558,11 +558,14 @@ export class Runtime {
...
    * @param program TEAL code
    * @param args arguments passed
+   * @returns logic signature account with arguments.
```
Comment (ratik21) [inline packages/runtime/src/runtime.ts:566]:
```suggestion
   * @returns logic signature with arguments.
```

**Review by ratik21 — APPROVED** [review]

File: examples/bond/test/bond-token-flow.js (hunk 1 above)
Comment (robert-zaremba) [reply examples/bond/test/bond-token-flow.js:110]: Good catch, in other places we are using `Lsig`, so I will rename it here as well.

Comment (robert-zaremba) [reply examples/bond/test/bond-token-flow.js:110]: renamed.

**Pull request merged**